.azure/pipelines/ci.yml

```diff
@@ -58,75 +58,62 @@ jobs:
     # if they have already been signed. This results in slower builds due to re-submitting the same .nupkg many times for signing.
     # The sign settings have been configured to
 
-    - script: ./eng/scripts/cibuild.cmd
+    - script: ./build.cmd
+          -ci
           -arch x64
-          -BuildNative
-          /p:DisableCodeSigning=true
+          -pack
+          -all
+          -buildNative
           /bl:artifacts/log/build.x64.binlog
           $(_BuildArgs)
       displayName: Build x64
     # TODO: make it possible to build for one Windows architecture at a time
     # This is going to actually build x86 native assets. See https://github.com/aspnet/AspNetCore/issues/7196
 
     # Build the x86 shared framework
-    # Set DisableSignCheck because we'll run sign check in an explicit step after installers build
-    - script: ./eng/scripts/cibuild.cmd
+    - script: ./build.cmd
+            -ci
             -arch x86
-            -NoRestore
```

Comment:
Hmm I might want to investigate this separately. Specifically because I know there were some changes with lifetimes of targets and interleaving of steps introduced with converting to arcade so these changes may be deliberate. Since I wouldn't want to hold up the PR and this is more of an optimization, I would do this as a follow up.